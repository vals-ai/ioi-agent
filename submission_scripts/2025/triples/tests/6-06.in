version https://git-lfs.github.com/spec/v1
oid sha256:b54f3018e7304a52b875cece12c27fcd0dafca6ac224c3998ad699baedaa4994
size 1177822
