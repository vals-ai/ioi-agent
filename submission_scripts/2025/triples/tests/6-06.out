version https://git-lfs.github.com/spec/v1
oid sha256:157acc9b9da7b25b8b3c55e5247aacf7f66d87d12b8c6a95526838023db4e7dc
size 43
